Libs/MRML/Core/vtkMRMLSegmentationNode.cxx

```diff
@@ -460,8 +460,16 @@ void vtkMRMLSegmentationNode::GetRASBounds(double bounds[6])
     {
     this->Segmentation->GetBounds(bounds);
     }
+
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLSegmentationNode::GetNodeBounds(double bounds[6])
+{
```

Comment:
Actually, GetRASBounds was incorrectly implemented for vtkMRMLSegmentationNode.

this->Segmentation->GetBounds(bounds) => this should be vtkMRMLSegmentationNode::GetBounds() result.

vtkMRMLSegmentationNode::GetRASBounds() can be computed as usual, by transforming GetBounds() result to RAS.